co-simulation/fed/mosaic-infrastructure/src/test/java/org/eclipse/mosaic/fed/infrastructure/ambassador/InfrastructureInstanceManagerTest.java

```diff
@@ -149,7 +149,7 @@ public void testOnDetectedObject() throws IOException{
                 0.5,
                 "sensor6",
                 "projection String",
-                "Object1",
+                101,
```

Comment:
Is this correct? Original one is using "Object1" which seems to correlate with 100